roles/installer/defaults/main.yml

```diff
@@ -158,6 +156,9 @@ ee_images:
 
 _control_plane_ee_image: quay.io/ansible/awx-ee:latest
 
+_init_container_image: "{{ _control_plane_ee_image.split(':')[0] }}"
+_init_container_image_version: "{{ _control_plane_ee_image.split(':')[1] }}"
```

Comment:
I wonder if downstream the control plane ee has everything to run all the different init scripts we could run in this init container